app.py

```diff
@@ -204,12 +211,67 @@ def tootfeed(query_feed):
         for toot in buffered:
 
             text = BeautifulSoup(toot['content'], "html.parser").text
+            pubdate = toot['created_at']
+            if not pubdate.tzinfo:
```

Comment:
Similarly, the `created_at` datetime already has timezone info.  I'm not entirely sure which timezone is used, but I suspect it is whatever my user account's preference is set to.